Gemfile

```diff
@@ -10,7 +10,11 @@ gem "guard", "~> 2.14"
 gem "guard-rspec", "~> 4.7"
 gem "guard-rubocop", "~> 1.2"
 gem "listen", "3.0.7"
-gem "pry", "~> 0.10"
+if Gem::Version.create(RUBY_VERSION) < Gem::Version.create("2.4.0")
```

Comment:
For pass CI.
But, It might be better to upgrade required ruby version to >= 2.4.0.